config.yaml

```diff
@@ -13,8 +13,14 @@ module:
     docker:
         start:
         labels:
-            description: "I dont know"
-            io.k8s.description: "I dont know too"
+            com.redhat.component: "https://github.com/fedora-modularity/base-runtime"
```

Comment:
please put a comment here saying this is a short-term solution since there is no bugzilla component yet